api/src/test/java/org/openmrs/validator/VisitValidatorTest.java

```diff
@@ -448,4 +448,52 @@ public void validate_shouldFailValidationIfFieldLengthsAreNotCorrect() {
 		new VisitValidator().validate(visit, errors);
 		assertEquals(true, errors.hasFieldErrors("voidReason"));
 	}
+	
+	/**
+	 * @see VisitValidator#validate(Object,Errors)
+	 */
+	@Test
+	public void validate_shouldFailValidationIfVisitDateBeforeBirthDate() {
+		VisitService vs = Context.getVisitService();
+		Visit visit = new Visit();
+		
```

Comment:
remove newline between visit initialization and all the setters.

to get a style of arrange (setup what you need for the test), act (execute the method/thing under test), assert (AAA). this makes it easy to read and spot what you are testing